src/components/BreakoutRoomsEditor/SelectableParticipant.vue

```diff
@@ -139,7 +139,13 @@ export default {
 		t,
 
 		handleEnter(event) {
-			event.target.checked = !event.target.checked
+			if (this.isBulkSelection) {
+				event.preventDefault()
+				event.stopPropagation()
```

Comment:
To simplify, we can put both to `v-on`. We don't need to prevent `preventDefault` and `stopPropagation` in the second case.